editor/app/layout.tsx

```diff
@@ -1,7 +1,23 @@
 import "../styles/global.scss";
+import { Metadata } from "next";
 import AuthProvider from "@/contexts/Auth";
 import SidebarLayout from "@/components/SidebarLayout";
 
+export const metadata: Metadata = {
+  title: "Vision Zero Editor",
+  description: "Austin's crash data management platform",
+  authors: [
+    {
+      name: "Austin Transportation & Public Works Data & Technology Services",
+      url: "https://austinmobility.io/",
+    },
+  ],
+  icons: {
+    icon: `${process.env.NEXT_PUBLIC_BASE_PATH || ""}/favicon.ico`,
+  },
+  keywords: ["Vision Zero", "Austin", "City of Austin", "Safety", "Civic Tech"],
```

Comment:
I thought maybe "Texas" should be added here, but if you try searching Austin Minnesota vision zero, our stuff comes up anyway